### ISSUE TITLE
Issues with testing against `pyuvsim`

### ISSUE THREAD

**Issue by tyler-a-cox** [tyler-a-cox/fftvis#9]
I'm having difficulty testing against `pyuvsim` due to `mpi4py` issues popping up in specific versions of python. `matvis` seemingly doesn't have these same issues despite have a similar setup.

**Comment by kartikmandar** [comment]
Is this issue still there?

**Comment by tyler-a-cox** [comment]
No, we can just go ahead and close this. This was just an environment setup issue. Thanks for checking!
